js/pwindow.js

```diff
@@ -0,0 +1,266 @@
+    n.addEventListener('click', (e) => {
+      const { targetId } = e;
+      const tdetail = tdetail[targetId];
+    });
+  });
+
+  tdetail[i].technologies.forEach((tech, j) => {
+    const tag = document.createElement('li');
+    URL.append(tag);
+    tag.classList.add(tech);
+    tag.innerText = tdetail[i].technologies[j];
+  });
+
+  const bigContainer = document.querySelector('.works');
+  const oneCard = document.createElement('div');
+  oneCard.innerHTML = `<img class="dtcards-pic" src="${element.img}" alt="Snapshoot Portfolio 1">
+    <img class="mbcards-pic" src="${element.img2}" alt="image">
+    <div class="left-block">
+      <h2 class="project-title">${element.name}</h2>
+      <div class="frame-2">
+          <ul>
+              <li class="client"> CANOPY</li>
+              <li><a href="#"><img class="counter" src="Images/Counter.png" alt="dot\fulstop"></a></li>
+              <li class="role">Back End Dev</li>
+              <li><a href="#"><img class="counter" src="Images/Counter.png" alt="dot\fullstop"></a></li>
+              <li class="year-label">2015</li>
+          </ul>
+      </div>
+      <p>
+         A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <div class="tags">
+        <ul>
+          <li class="tag">${element.technologies[0]}</li>
+          <li class="tag">${element.technologies[1]}</li>
+          <li class="tag">${element.technologies[2]}</li>
+        </ul>
+      </div>
+      <div class="abutton">
+        <button type="button" id="${i}">See Project</button>
+      </div>
+    </div>`;
+  oneCard.classList.add('single-card');
+  oneCard.classList.add('card1');
+  bigContainer.appendChild(oneCard);
+  const button = document.createElement('button');
+
+  button.classList.add('showBtn');
+  button.setAttribute('id', i);
+  button.innerText = 'See Project';
+});
+
+tdetail[i].technologies.forEach((tech, j) => {
+  const tag = document.createElement('li');
+  URL.append(tag);
+  tag.classList.add(tech);
+  tag.innerText = tdetail[i].technologies[j];
+});
+
+// projects.forEach((element, i) => {
+//   const works = document.querySelector('.works');
+//   const section = document.createElement('section');
+//   const image = document.createElement('img');
+//   const img = document.createElement('img');
+//   const img1 = document.createElement('img');
+//   const div = document.createElement('div');
+//   const miniDiv = document.createElement('div');
+//   const h4 = document.createElement('h4');
+//   const h41 = document.createElement('h4');
+//   const ul = document.createElement('ul');
+//   const button = document.createElement('button');
+//   const h1 = document.createElement('h1');
+//   const p = document.createElement('p');
+//   const a = document.createElement('a');
+
+//   // working on image in the card
+//   image.src = element.img;
+//   image.classList.add('cards-pic');
+//   image.setAttribute('alt', 'Portfolio first card');
+
```

Comment:
Great job team 🤝

- [ ] It is regarded as a bad practice according to the [Javascript good practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) to leave commented code in your project. Kindly remove commented and unused code from your javascript files.